tests/app/helpers/test_template_helpers.py

```diff
@@ -64,114 +161,299 @@ def test_get_footer_context_nisra_theme(app):
+        ("census", "en", "census"),
+        ("census", "cy", "census"),
+        ("census-nisra", "en", "census"),
+    ],
+)
+def test_correct_theme_in_config(app: Flask, theme: str, language: str, expected: str):
+    with app.app_context():
+        survey_config = get_survey_config(theme=theme, language=language)
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["theme"]
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    "theme,language,expected",
+    [
+        ("default", "en", None),
+        ("business", "en", None),
+        ("health", "en", None),
+        ("social", "en", None),
+        ("northernireland", "en", None),
+        ("census", "en", "Census 2021"),
+        ("census", "cy", "Census 2021"),
+        ("census-nisra", "en", "Census 2021"),
+    ],
+)
+def test_correct_survey_title_in_config(
+    app: Flask, theme: str, language: str, expected: str
+):
+    with app.app_context():
+        survey_config = get_survey_config(theme=theme, language=language)
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["survey_title"]
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    "theme,language,expected",
+    [
+        ("default", "en", []),
+        ("business", "en", []),
+        ("health", "en", []),
+        ("social", "en", []),
+        ("northernireland", "en", []),
+        ("census", "en", [{"nisra": False}]),
+        ("census", "cy", [{"nisra": False}]),
+        ("census-nisra", "en", [{"nisra": True}]),
+    ],
+)
+def test_correct_data_layer_in_config(
+    app: Flask, theme: str, language: str, expected: str
+):
+    with app.app_context():
+        survey_config = get_survey_config(theme=theme, language=language)
 
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["data_layer"]
+    assert result == expected
 
-def test_get_census_base_url_welsh_is_priority_over_nisra():
-    expected = "https://cyfrifiad.gov.uk/"
-    result = get_census_base_url(schema_theme="census-nisra", language_code="cy")
 
-    assert expected == result
+def test_include_csrf_token_false(app: Flask):
+    expected = False
 
+    with app.app_context():
+        survey_config = SurveyConfig()
 
-def test_get_census_base_url_ga():
-    expected = "https://census.gov.uk/"
-    result = get_census_base_url(schema_theme="census", language_code="ga")
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=False,
+            survey_config=survey_config,
+        ).context["include_csrf_token"]
```

Comment:
We should really test `include_csrf_token` being `True` here as well.

Reply:
Yep - I thought it was tested above, good spot